#!/usr/bin/env python3

PACKAGE = "siemens_plc"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("laser_config_1", bool_t, 0, "Laser config parameter 1", True)
gen.add("laser_config_2", str_t, 0, "Laser config parameter 2", "config_2")
gen.add("laser_config_3", double_t, 0, "Laser config parameter 3", 5.0, 0.0, 10.0)

exit(gen.generate(PACKAGE, "siemens_plc", "laser_config_param"))
